dev/docker/ballista-base.dockerfile

```diff
@@ -23,7 +23,7 @@
 
 
 # Base image extends debian:buster-slim
-FROM rust:1.51.0-buster AS builder
+FROM rust:1.52.1-buster AS builder
```

Comment:
1.53 is out now

Reply:
i have 2 more PRs open. i'll change it in one of the other ones, if that's ok